ipv-core/src/main/java/com/ibm/whc/deid/providers/masking/BinningMaskingProvider.java

```diff
@@ -74,4 +87,43 @@ public String mask(String identifier) {
 
     return String.format(this.format, Long.valueOf(lowerBase), Long.valueOf(higherBase));
   }
+  
+  public String maskConstant(String identifier) {
```

Comment:
The new logic needs to be triggered from **public String mask(String identifier)**, as that is how this provider is going to be called by the system.  You could probably check the new config parameters as soon as the "value" variable is set.